src/utils/ICS24Host.sol

```diff
@@ -163,4 +184,17 @@ library ICS24Host {
         merklePrefix[merklePrefix.length - 1] = abi.encodePacked(merklePrefix[merklePrefix.length - 1], path);
         return merklePrefix;
     }
+
+    /// @notice Convert a uint64 to big endian bytes representation
+    /// @param value The uint64 value
+    /// @return The big endian bytes representation
+    function uint64ToBigEndian(uint64 value) internal pure returns (bytes8) {
```

Comment:
I wanna understand better why we need this function and:
1. If we can get rid of this by using little endian in ibc-go or some other change in ibc-go
2. If we can find an external library that has this audited function. (Not super needed)

Reply:
This is the reasoning I believe: https://github.com/cosmos/ibc-go/issues/7132

1. Lets ask speccers about it
2. I have not been able to find a library that does this